charms/kfp-api/src/charm.py

```diff
@@ -40,18 +42,19 @@ def __init__(self, *args):
 
         self.prometheus_provider = MetricsEndpointProvider(
             charm=self,
-            relation_name="monitoring",
+            relation_name="metrics-endpoint",
             jobs=[
                 {
-                    "job_name": "ml_pipeline",
-                    "scrape_interval": self.config["metrics-scrape-interval"],
-                    "metrics_path": self.config["metrics-api"],
+                    "metrics_path": METRICS_PATH,
                     "static_configs": [{"targets": ["*:{}".format(self.config["http-port"])]}],
                 }
             ],
         )
 
-        self.dashboard_provider = GrafanaDashboardProvider(self)
+        self.dashboard_provider = GrafanaDashboardProvider(
```

Comment:
don't need to change this line now that we are using the default name for the relation

Reply:
Yeah I think what @agathanatasha is suggesting is to do this:
```suggestion
        self.dashboard_provider = GrafanaDashboardProvider(self)
```